src/main/java/com/xceptance/neodymium/NeodymiumRunner.java

```diff
@@ -202,6 +238,21 @@ protected Object createTest() throws Exception
             testMethods.addAll(buildCrossProduct(testAnnotatedMethod.getMethod(), builderList, builderDataList));
         }
 
+        // filter test methods by regex
+        String testExecutionRegex = Neodymium.configuration().getTestNameFilter();
+        if (testExecutionRegex != null)
```

Comment:
Please use `StringUtils.isNotEmpty(testExecutionRegex)` instead since a contains of "" would also result in no filtering but would cost some computation time.

Reply:
Done